main.pm

```diff
@@ -24,6 +24,7 @@ sub load_install_tests() {
     # interaction
     return 1 if get_var('OPENQA_FROM_GIT');
     loadtest 'install/openqa_worker' unless get_var('OPENQA_FROM_BOOTSTRAP');
+    loadtest 'install/worker_ay_validation' if get_var('VALIDATE_AUTOYAST');
```

Comment:
Would be awesome to see a validation run 🙃

Reply:
Can you access http://10.100.224.110/tests/62#step/worker_ay_validation/12?

Reply:
I'm afraid I can't

Reply:
https://openqa.opensuse.org/tests/4415391